tests/test-core/utils/js/ConditionalTestUtils.js

```diff
@@ -100,13 +100,44 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.test.conditionalTestUtils.isBrowserOnWindows = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Windows", "{fluid.browser.platformName}");
+        return fluid.test.conditionalTestUtils.contextValueContains("Win", "{fluid.browser.platformName}");
```

Comment:
Could we have a sketch of a few possible platformName strings which include "Win" but not "Windows" but are on Windows? Or a stackOverflow reference or similar